helpers/argumentParsers.go

```diff
@@ -579,13 +581,39 @@ const (
 	PR_SET_FP_MODE
 	PR_GET_FP_MODE
 	PR_CAP_AMBIENT
+	_
+	_
 	PR_SVE_SET_VL
 	PR_SVE_GET_VL
 	PR_GET_SPECULATION_CTRL
 	PR_SET_SPECULATION_CTRL
 	PR_PAC_RESET_KEYS
 	PR_SET_TAGGED_ADDR_CTRL
 	PR_GET_TAGGED_ADDR_CTRL
+	PR_SET_IO_FLUSHER
+	PR_GET_IO_FLUSHER
+	PR_SET_SYSCALL_USER_DISPATCH
+	PR_PAC_SET_ENABLED_KEYS
+	PR_PAC_GET_ENABLED_KEYS
+	PR_SCHED_CORE
+	PR_SME_SET_VL
+	PR_SME_GET_VL
+	PR_SET_MDWE
+	PR_GET_MDWE
+	PR_SET_MEMORY_MERGE
+	PR_GET_MEMORY_MERGE
```

Comment:
Some of those were already inserted in Tracee's: https://github.com/aquasecurity/tracee/blob/1150c947b726ba22f0432106d85abc389ffb8699/pkg/events/parsers/data_parsers.go#L496-L502

Other were ignored due to from unsupported archs.

Reply:
Ok, we've discussed offline, merging it for now.